packages/fast-components-react-msft/src/paragraph/paragraph.spec.tsx

```diff
@@ -66,4 +58,11 @@ describe("paragraph", (): void => {
 
         expect(rendered.prop("size")).toBe(ParagraphSize._3);
     });
+
+    test("should accept and render children", () => {
+        const rendered: any = shallow(<MSFTParagraph>Children</MSFTParagraph>);
+
+        expect(rendered.prop("children")).not.toBe(undefined);
```

Comment:
minor nit, but you don't really need both of these

Reply:
Will remove